src/Swashbuckle.AspNetCore.SwaggerGen/SchemaGenerator/OpenApiSchemaExtensions.cs

```diff
@@ -183,8 +183,15 @@ private static void ApplyRangeAttribute(OpenApiSchema schema, RangeAttribute ran
         {
 #if NET8_0_OR_GREATER
 
-            schema.ExclusiveMinimum = rangeAttribute.MinimumIsExclusive;
-            schema.ExclusiveMaximum = rangeAttribute.MaximumIsExclusive;
+            if(rangeAttribute.MinimumIsExclusive)
```

Comment:
```suggestion
            if (rangeAttribute.MinimumIsExclusive)
```

Reply:
I do sorry!. Usually VS formats the code, but it does not seem to format it inside Compile conditions